skyportal/tests/api/test_annotations_on_spectrum.py

```diff
@@ -48,7 +48,7 @@ def test_add_and_retrieve_annotation_group_id(
         token=annotation_token,
     )
 
-    assert status == 400
+    assert status in [400, 401]
```

Comment:
Why 400 and not 401?

Reply:
Nevermind I see both options.

Reply:
This should be one or the other, I suppose.